docs/testem-component.md

```diff
@@ -0,0 +1,87 @@
+| `testPages`               | `{Array}`   | One or more test pages to load in the browser, relative to the directory in which your configuration file is stored (see "paths" below). |
+| `serveDirs`               | `{Array}`   | One or more directories to host within the Testem environment. |
+| `testemOptions`           | `{Object}`  | The raw configuration options to pass to Testem.  See [the Testem docs](https://github.com/testem/testem/blob/master/docs/config_file.md) for supported options. |
+| `cleanup.initial`         | `{Array}`   | An array of cleanup definitions (see below) to be cleaned up before the tests are run. |
+| `cleanup.final`           | `{Array}`   | An array of cleanup definitions (see below) to be cleaned up after the tests are run and all reporting is complete. |
+
+## Cleanup Definitions
+
+The initial and final cleanup options expect to be passed an array of "cleanup definitions", which support the following
+attributes:
+
+| Option            | Type         | Description                           |
+| ----------------- | ------------ | ------------------------------------- |
+| `name` (required) | `{String}`   | A "nickname" to use for this directory in log messages. |
+| `path` (required) | `{String}`   | The path to remove. |
+| `isTestemContent` | `{Boolean}`  | If this directory contains testem-generated content, we use a different cleanup method. |
+
+# Component Invokers
+
+## `{gpii.testem}.handleTestemStart(config, data, callback)`
+* `config`: The configuration information Testem exposes as part of its lifecycle.
+* `data`: The data Testem exposes as part of its lifecycle.
+* `callback`: A function to be called when it is safe for Testem to run tests.  If you do not call this callback, Testem will hang indefinitely before running tests.
+* Returns: Nothing.
+
+An invoker which is called before testem begins its test run.  Starts a chain of events which cleanup before the tests, instrument code, start test fixtures, and perform
+other preparatory work.  For details on how this works, see ["The Testem Event Lifecycle"](testem-lifecycle.md).
+
+## `{gpii.testem}.handleTestemExit(config, data, callback)`
+* `config`: The configuration information Testem exposes as part of its lifecycle.
+* `data`: The data Testem exposes as part of its lifecycle.
+* `callback`: A function to be called when it is safe for Testem to exit.  If you do not call this callback, you will not be able to quit Testem without killing the process.
+* Returns: Nothing.
+
+An invoker which is called when Testem has completed all tests.  Used to stop test fixtures, prepare reports, and remove
+temporary content.  For details on how this works, see ["The Testem Event Lifecycle"](testem-lifecycle.md).
+
+## `{gpii.testem}.getTestemOptions()`
+
+An invoker which exposes the final Testem options in a way that Testem can read from a configuration file.  See
```

Comment:
The language "in a way that Testem can read" doesn't quite make sense to me. Is it meant to express that the options take the same form that Testem might read from a configuration file, or that it will do so?